cns/restserver/homeazmonitor.go

```diff
@@ -144,17 +144,33 @@ func (h *HomeAzMonitor) Populate(ctx context.Context) {
 
 // update constructs a GetHomeAzResponse entity and update its cache
 func (h *HomeAzMonitor) update(code types.ResponseCode, msg string, homeAzResponse cns.HomeAzResponse) {
-	log.Debugf(msg)
 	resp := cns.GetHomeAzResponse{
 		Response: cns.Response{
 			ReturnCode: code,
 			Message:    msg,
 		},
 		HomeAzResponse: homeAzResponse,
 	}
+	h.logHomeAzResponse(resp)
 	h.updateCacheValue(resp)
 }
 
+// logHomeAzResponse logs HomeAz Response
+func (h *HomeAzMonitor) logHomeAzResponse(resp cns.GetHomeAzResponse) {
```

Comment:
Hey Zetao, thinking more on this, I think we should log the HomeAzResponse everytime ( not just when it gets updated ). That's because, we may lose the AZ log if it doesn't change ( which it shouldn't ) for long time. Having that log would be beneficial.

Reply:
that is right. I will update this

Reply:
we don't need the func body below. We should just print the resp struct here.

Reply:
but in this way we can log it as error when there have errors in refreshing home az

Reply:
I would keep the logging simple. Since we already have logic in DNC to identify mismatch on the home AZ we can add alerting there